lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -25,7 +25,7 @@ def analyze(changes)
       capture3!("git", "commit-graph", "write", "--reachable", "--changed-paths", "--no-progress")
 
       target_files = Pathname.glob("**/*", File::FNM_DOTMATCH).filter do |path|
-        path.file? && !path.fnmatch?("**/.git/*", File::FNM_DOTMATCH)
+        path.file? && !path.fnmatch?(".git/*", File::FNM_DOTMATCH) && !path.fnmatch?("*/.git/*", File::FNM_DOTMATCH)
```

Comment:
[suggestion] How about this? ⬇️ 

```suggestion
        path.file? && !path.fnmatch?(".git/**", File::FNM_DOTMATCH)
```